### ISSUE TITLE
Allow to set php.ini values per application

### ISSUE THREAD

**Issue by bbak** [nginx/unit#98]
As soon as I need multiple Applications of type php (with the same php-Version/Engine) that require different settings in php.ini, I basically cannot use Unit since there's no method to pass those settings to the embedded php library.

In Apache mod_php it can be done via .htaccess, PHP-FPM reads a `.user.ini` and accepts php.ini settings in the configuration of the FPM-Pool. 
IMHO Unit needs something similar, preferably as part of the Built-In configuration system.

**Comment by avkarenow** [comment]
Good if it also can load other extensions for every App or even whole php.ini config can be used with every App (change the variables with PHP_INI_SYSTEM flag).

.user.ini file is good for users without access to server configuration and it should allow changes only variables with PHP_INI_PERDIR/PHP_INI_ALL flags.
Now my users are using .user.ini to reconfigure their applications (php-fpm), so it will good to still have that possibility with Unit.

**Comment by VBart** [comment]
Support for configuring php.ini and individual parameters has been added by https://github.com/nginx/unit/commit/8f278a5fedaa570398318dcec77eea7d8c856796, https://github.com/nginx/unit/commit/388390888bc0c2a3589b71e5b3dc57408a5f4c44, and now released with Unit 1.2.

Here's an example:
```json
{
    "type": "php",
    "root": "/www/site",
    "script": "phpinfo.php",

    "options": {
        "file": "/path/to/php.ini",
        "admin": {
            "memory_limit": "256M",
            "variables_order": "EGPCS",
            "short_open_tag": "1"
        },
        "user": {
            "display_errors": "0"
        }
    }
}
```
